### ISSUE TITLE
my issues

### ISSUE THREAD

**Issue by phucle1112** [phucle1112/week2#1]
[week-1-lists-phucle1112.zip](https://github.com/phucle1112/week2/files/2646180/week-1-lists-phucle1112.zip)

![capturew2](https://user-images.githubusercontent.com/43037470/49473564-ae2bb580-f812-11e8-81e9-8a039ed4c99b.PNG)
# issues
@zaldawid dear sir, i already committed for this assignment, however, i got some problems and i need your help: Do i have to base on the single linked code to fix the double linked code ? When i build the code, a lot of error about the operator? can you suggest me what are some posibility error ? if the "_tail" does not need in the double linked , what esle can i use ? "_node" or something else ? I am sorry because maybe my questions are really stupid, but i really need your help.

**Comment by zaldawid** [comment]
What you are trying to do is illegal and the compiler complains about it.
_node is of type node_t and you are trying to assign a pointer to it (node_t*). This won't work.
Besides, you should never assign anything to _node in your implementation. You might change the _next or _prev pointers of the _node but not _node itself. (I suspect that's what you wanted to do)

**Comment by zaldawid** [comment]
And next time, commit and push your code to Github. Don't zip the project and attach it.

**Comment by zaldawid** [comment]
And don't be sorry about asking questions. If anything be sorry for not asking them on time.

**Comment by phucle1112** [comment]
@zaldawid thanks sir. I will try it again

**Comment by phucle1112** [comment]
@zaldawid dear sir, i re-committed the project again but this is what i saw
[==========] Running 26 tests from 7 test cases.
[----------] Global test environment set-up.
[----------] 1 test from list_basic
[ RUN      ] list_basic.should_always_pass
[       OK ] list_basic.should_always_pass (0 ms)
[----------] 1 test from list_basic (6 ms total)

[----------] 8 tests from list_constructors
[ RUN      ] list_constructors.default_ctor
[       OK ] list_constructors.default_ctor (0 ms)
[ RUN      ] list_constructors.initializer_list_type

Process finished with exit code -1073741819 (0xC0000005)

i think there are somethings still wrong here, do you have any suggestions?

**Comment by zaldawid** [comment]
That's correct. Because the implementation of list is incomplete the tests will crash (so you won't even see all of them executing). You'll have to fix the list functions used in each test one by one to see more tests executed. Also don't forget to fix the list destructor.

**Comment by phucle1112** [comment]
Thanks for your suggestion. I will fix them

**Comment by phucle1112** [comment]
@zaldawid i guess i only have to fix only on the list.h file, right ?
